testsuite/MDAnalysisTests/converters/test_rdkit.py

```diff
@@ -484,82 +498,188 @@ def test_set_atom_property(self, attr, value, getter):
+        ("dummy_reactant", "dummy_product"),
     ], indirect=["rdmol", "product"])
-    def test_reassign_props_after_reaction(self, rdmol, product, name):
-        _reassign_props_after_reaction(rdmol, product)
+    def test_reassign_index_after_reaction(self, rdmol, product):
+        _reassign_index_after_reaction(rdmol, product)
         atom = product.GetAtomWithIdx(0)
-        if name == "props":
-            assert atom.GetProp("foo") == "bar"
-            assert atom.GetIntProp("_MDAnalysis_index") == 1
-            assert atom.GetDoubleProp("_MDAnalysis_charge") == 4.2
-            assert atom.GetProp("_MDAnalysis_type") == "C.3"
-            with pytest.raises(KeyError, match="old_mapno"):
-                atom.GetIntProp("old_mapno")
-            with pytest.raises(KeyError, match="react_atom_idx"):
-                atom.GetUnsignedProp("react_atom_idx")
-        elif name == "noprops":
-            with pytest.raises(KeyError, match="old_mapno"):
-                atom.GetIntProp("old_mapno")
-            with pytest.raises(KeyError, match="react_atom_idx"):
-                atom.GetUnsignedProp("react_atom_idx")
-        elif name == "nomap":
-            with pytest.raises(KeyError, match="react_atom_idx"):
-                atom.GetUnsignedProp("react_atom_idx")
-            with pytest.raises(KeyError, match="_MDAnalysis_index"):
-                atom.GetIntProp("_MDAnalysis_index")
-
-    @pytest.mark.parametrize("smi_in", [
+        assert atom.GetIntProp("_MDAnalysis_index") == 1
+
+    @pytest.mark.parametrize("smi", [
         "c1ccc(cc1)-c1ccccc1-c1ccccc1",
         "c1cc[nH]c1",
-        "c1ccc(cc1)-c1ccc(-c2ccccc2)c(-c2ccccc2)c1-c1ccccc1",
-        "c1ccc2c(c1)c1ccccc1c1ccccc1c1ccccc1c1ccccc21",
-        "c1csc(c1)-c1ccoc1-c1cc[nH]c1",
-        "C1=C2C(=NC=N1)N=CN2",
-        "CN1C=NC(=C1SC2=NC=NC3=C2NC=N3)[N+](=O)[O-]",
-        "c1c[nH]c(c1)-c1ccc(s1)-c1ccoc1-c1c[nH]cc1-c1ccccc1",
-        "C=CC=CC=CC=CC=CC=C",
-        "NCCCCC([NH3+])C(=O)[O-]",
-        "CC(C=CC1=C(C)CCCC1(C)C)=CC=CC(C)=CC=[NH+]C",
-        "C#CC=C",
-        # HID HIE HIP residues, see PR #2941
-        "O=C([C@H](CC1=CNC=N1)N)O",
-        "O=C([C@H](CC1=CN=CN1)N)O",
         "O=C([C@H](CC1=C[NH1+]=CN1)[NH3+])[O-]",
     ])
-    def test_order_independant(self, smi_in):
+    def test_transfer_properties(self, smi):
+        mol = Chem.MolFromSmiles(smi)
+        mol = self.mol_to_template(mol)
+        old = {}
+        for atom in mol.GetAtoms():
+            ix = atom.GetIdx()
+            atom.SetIntProp("_MDAnalysis_index", ix)
+            atom.SetProp("dummy", f"foo_{ix}")
+            old[ix] = {"_MDAnalysis_index": ix, "dummy": f"foo_{ix}"}
+        _infer_bo_and_charges(mol)
+        newmol = _standardize_patterns(mol)
+        new = {}
+        for a in newmol.GetAtoms():
+            ix = a.GetIntProp("_MDAnalysis_index")
+            new[ix] = {"_MDAnalysis_index": ix,
+                       "dummy": a.GetProp("dummy")}
```

Comment:
```suggestion
                       "dummy": a.GetProp(f"foo_{ix}")}
```
Otherwise I'm not sure this tests much.

Reply:
Nope the test is fine, the name of the property is "dummy" and its value should be "foo_{ix}", so to retrieve the value we just call `atom.GetProp("dummy")`